apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -290,9 +290,9 @@
                     <!-- region bank ledgers-->
                     <div class="clearfix"></div>
                     <div *ngIf="!isMobileScreen">
-                        <h1 class="mr-t2 primary-color" *ngIf="isBankTransactionLoading">
+                        <h1 class="mr-t2 primary-color" [hidden]="!isBankTransactionLoading">
```

Comment:
why are we doing this?

Reply:
We are doing this as when the user refreshes the E-bank transactions then the whole DOM is recreated again which takes a long time even if the user has't changed any transaction which makes the UI slow